pages/linux/xbps.md

```diff
@@ -0,0 +1,23 @@
+# xbps
+
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+
+- Install packages; Option -S --synchronize:
+
+`xbps-install --synchronize {{package_name1}} {{package_name2}}`
+
+- Search for a package; Option -R --repository; Mode -s --search; Shorthand -Rs:
+
+`xbps-query --repository -s {{package_name}}`
```

Comment:
-R can be misinterpreted as --recursive but in xbps-query case its --repository, so again, I am leaving the long version; -s stands for --search, I'm on the edge of changing it to the long version too.

Reply:
As long as you have a word in the description which starts with the flag, it is fine.

Reply:
So should I leave it as it is right now or return it back to "xbps-query -Rs" while there is a description for every flag and option involved? Kinda confused by now.

Reply:
I can do it myself if you are okay with it.

Reply:
yeah sure np